index.md

```diff
@@ -72,3 +72,84 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
 ## UC15 Verified Impact
 
 Lynn is a verification agent for REC2022. She is called by HydroElec to audit their new production facility. After reviewing the certified proposal, she visits the HydroElec site to inspect and verify both the capacity of the system under operating conditions and the security of the automated smart meters. She uses a standard set of criteria established by the UN to record her observations and make a determination about the project’s compliance with both its certified proposal and UN requirements. After finding everything in order, Lynn issues a verification credential with a six month expiry and a maximum capacity of 20 billion kWh, which she gives to HydroElec. The credential references both the original proposal and its certification using machine-verifiable identifiers to ensure the authenticity of the verification.
+
+# Focal Use Cases
+
+# Shang-chi Proves Impact
+## Background
+Shang-Chi is the Manager of a Hong Kong social enterprise - EldHealth focusing on elderly physical health. Over the past years, EldHealth has expanded rapidly in its operation and has many beneficiaries enrolled in its signature program, iHealth. In order to get more funding, Shang-Chi needs to prove that beneficiaries have become healthier as a result of enrolling in iHealth. 
+
+However, the authenticity of the data report Shang-Chi has produced has received a lot of doubt from his new funders as there has not been a good working relationship and trust established between the two yet. 
+
+The new funders have shown low confidence in the authenticity of the data, which shows that 99.9% of the beneficiaries improved in health in only 6 months since enrolling in iHealth. Shang-Chi thus needs to ensure the organization’s impact accountability to its funders. The data (reported by the beneficiaries) against the intended outcomes need to be validated. 
+
+To do so, the manager uses a platform to allow him to ensure the authenticity of the data submitter (beneficiary) and know the time and date of the data submission. The manager also has the ability to validate the credibility of the outcome data based on external evidence (e.g. photos, documents - offchain data). 
+
+## Scenario
+
+Shang-Chi collects data about the health of beneficiaries who joined iHealth. He sends standard survey forms through the platform to target beneficiaries weekly, which data are then submitted back to Shang-Chi with evidence attached (e.g. photos of blood pressure records and weight measurement). Having all this information associated with the data, Shang-Chi then evaluates the data submitted and approves each submission record by signing on it (digitally). These validated data are incorporated into the reports to funder, which will be auditable, accountable, and traceable.
```

Comment:
"...platform to target beneficiaries weekly, which data are"

Should it be ", whose data"